src/graphics/command.cpp

```diff
@@ -12,10 +12,23 @@ void Command::set_card(PlayerCard *card) {
     if (!card)
         return;
     if (Action *action = dynamic_cast<Action *>(card); action) {
-        events.push_back(std::make_pair(Type::action, action));
+        events.push_back(std::make_pair(Type::action_on_hand, action));
         action->press();
     } else if (Minion *minion = dynamic_cast<Minion *>(card); minion) {
-        events.push_back(std::make_pair(Type::minion, minion));
+        events.push_back(std::make_pair(Type::minion_on_hand, minion));
+        minion->press();
+    }
+    check_commands();
+}
+
+void Command::set_shown_card(PlayerCard *card) {
+    if (!card)
+        return;
+    if (Action *action = dynamic_cast<Action *>(card); action) {
+        events.push_back(std::make_pair(Type::action_on_base, action));
```

Comment:
In current state, action cant be played on base, what is this for?

Reply:
For the future fitch
:)

Reply:
Okay